mne/commands/mne_flash_bem_model.py

```diff
@@ -23,8 +23,10 @@
 import os
 
 import mne
+from mne.utils import deprecated
```

Comment:
The permissions on this file changed not to be executable anymore, yet there's a `if is_main` at the bottom. Is this intended? It seems like you'd either want the `if is_main` *and* execute, or neither of them.